stock-filters/ChirpVillage/Block.py

```diff
@@ -1,10 +1,15 @@
 class Block(object):
 
-    def __init__(self, x, y, height):
+    # BLOCK TYPES
+    UNASSIGNED = 0
+    YARD = 1
+    BUILDING = 2
+    DOOR = 3
+    PATH = 4
+
+    def __init__(self, x, z, height):
```

Comment:
also should probably pass in type in the constructor, maybe as an optional parameter

Reply:
good point